python/lsst/ts/weatherstation/csc.py

```diff
@@ -157,6 +159,7 @@ async def end_disable(self, id_data):
         await self.wait_loop(self.telemetry_loop_task)
 
         await super().end_disable(id_data)
+        return SalRetCode.CMD_INPROGRESS
```

Comment:
Ack is already been sent in `begin_disable` no need to resend it here.

Reply:
Thanks for the clarification. I adjusted the code accordingly.